src/main/java/duke/Deadline.java

```diff
@@ -0,0 +1,60 @@
+package duke;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Class to maintain the deadline tasks entered by user.
+ */
+public class Deadline extends Task {
+    protected LocalDate deadline;
+
+    protected LocalTime endTime;
+
+    /**
+     * Constructor for Deadline class.
+     * @param list List to add
+     * @param deadline deadline
+     * @param endTime end time of deadline task
+     * @param type enum type
+     */
+

```

Comment:
Perhaps there should be no blank line between the documentation block and the method.